### DISCUSSION TITLE
'Sorting' event on DataGrid has column index but not sort direction

### DISCUSSION THREAD

**Discussion by TangDave** [AvaloniaUI/Avalonia#16689] [Q&A]
I'm trying to record the user's sorting preference on a DataGrid and I can get the column sorted by the user from the 'Sorting' event, but not the sort direction (i.e., ascending or descending). Am I using the wrong event for this?

**Comment by TangDave** [comment]
Just realized that this is trivial to track with a boolean flag changing when the event is triggered when the column doesn't change. Still might be useful to have the DataGridColumnEventArgs contain it somewhere.

**Reply by rabbitism** [reply]
Feel free to raise a pull request if you think this is necessary.
